charts/sn-platform/templates/prometheus/prometheus-statefulset.yaml

```diff
@@ -171,6 +171,9 @@ spec:
           - --web.enable-lifecycle
   {{ if .Values.prometheus.maxQuerySamples }}
           - --query.max-samples={{.Values.prometheus.maxQuerySamples }}
+  {{- end }}
+  {{ if .Values.prometheus.args.walCompressionEnabled }}
+          - --storage.tsdb.wal-compression
   {{- end }}
```

Comment:
Shall we also define a `.Values.prometheus.extraArgs` field? so we can custom args without updating chart next time.

Reply:
Thanks, I have fixed your comment, PTAL